app/uk/gov/hmrc/serviceconfigs/connector/NginxConfigConnector.scala

```diff
@@ -35,17 +35,18 @@ class NginxConfigConnector @Inject()(http: HttpClient, gitConf: GithubConfig, ng
 
   private val configKey = gitConf.githubApiOpenConfig.key
 
-  def configFor(env: String) : Future[Option[NginxConfigFile]] = {
+  def getNginxRoutesFile(fileName: String, environment: String) : Future[List[NginxConfigFile]] = {
```

Comment:
Does this ever return multiple config files or just one? Should it still be an Option?

Reply:
You are right. I changed it initially because I thought I would loop through the filenames and return the raw routes files but that didn't happen.